osu.Game.Rulesets.Osu/UI/OsuAnalysisSettings.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Game.Localisation;
+using osu.Game.Replays;
+using osu.Game.Rulesets.UI;
+using osu.Game.Screens.Play.PlayerSettings;
+
+namespace osu.Game.Rulesets.Osu.UI
+{
+    public partial class OsuAnalysisSettings : AnalysisSettings
```

Comment:
Rather than having settings exposed like this, it might make more sense (and reduce overheads) to use `[SettingSource]`.

Also as far as I can see, these settings aren't persisted anywhere?